.env.example

```
@@ -1,95 +1,20 @@
 
-# Logging
-LOG_LEVEL=INFO
-LOG_SIGNING_KEY=change_me_to_a_secure_random_string
-EXPO_PUBLIC_LOG_SIGNING_KEY=change_me_to_a_secure_random_string
-EXPO_PUBLIC_LOG_SALT=change_me_to_a_secure_random_string
-EXPO_PUBLIC_LOG_INGEST_URL=http://localhost:8080/api/ingest/logs
-EXPO_PUBLIC_LOG_LEVEL=INFO
+# Backend Configuration
+JWT_SECRET=your_jwt_secret_here_minimum_32_characters
+NODE_ENV=development
+CORS_ORIGIN=http://localhost:3000
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CORS_ORIGIN key should go before the JWT_SECRET key

(UnorderedKey)